tools/slang-unit-test/unit-test-vtable-stability.cpp

```
@@ -0,0 +1,1662 @@
+// (beyond `this`) is safe regardless of the declared parameter list.
+
+#include "slang.h"
+#include "unit-test/slang-unit-test.h"
+
+using namespace slang;
+
+// ---------------------------------------------------------------------------
+// Helper: call vtable slot `slot` on `obj`, passing only `this`.
+// ---------------------------------------------------------------------------
+static void callSlot(void* obj, int slot)
+{
```

Cppcheck (2.20.0)

[style] The function 'memcpy_s' is never used.

(unusedFunction)